src/Giraffe/EndpointRouting.fs

```diff
@@ -232,39 +270,48 @@ module Routers =
     let TRACE = applyHttpVerbToEndpoints TRACE
     let CONNECT = applyHttpVerbToEndpoints CONNECT
 
-    let route (path: string) (handler: HttpHandler) : Endpoint =
-        SimpleEndpoint(HttpVerb.NotSpecified, path, handler, id)
+    let route (path: string) (aspNetExtensions: AspNetExtension list) (handler: HttpHandler) : Endpoint =
```

Comment:
This is massive breaking change is it not?
Is means every existing code with `route` would break.

I'd go for the following:
```fsharp
let routeWithExtensions (path: string) (aspNetExtensions: AspNetExtension list) (handler: HttpHandler) : Endpoint = SimpleEndpoint(HttpVerb.NotSpecified, path, handler, id, aspNetExtensions)

let route (path: string) (handler: HttpHandler) : Endpoint = routeWithExtensions(path, [], handler)
```

Reply:
Indeed, it looks a better path to go